examples/sampleview/core/sampledesigner/DesignerScene.cpp

```diff
@@ -26,83 +27,40 @@
 #include "SampleViewFactory.h"
 #include "item_constants.h"
 #include "modelmapper.h"
+#include "modelutils.h"
 #include "sessionitem.h"
 #include <QGraphicsSceneMouseEvent>
 #include <QItemSelection>
 #include <QPainter>
 
 using namespace ModelView;
 
-namespace {
-class ModelCommandExecutor
+DesignerScene::DesignerScene(SampleModel* sample_model, QObject* parent)
+    : QGraphicsScene(parent)
+    , m_controller(*this, sample_model)
+    , m_selectionModel(nullptr)
+    , m_proxy(nullptr)
+    , m_block_selection(false)
+    , m_aligner(new SampleViewAligner(this))
+    , m_nodeEditor(new NodeEditor(parent))
```

Comment:
Memory leakage see I when parent nullptr is.

Reply:
Indeed, thanks! I will scan it with memcheck